### ISSUE TITLE
Improved support for screen reader: review description and localization

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/readium-desktop#3516]
In the app Settings, General section, the block related to "Improved support for screen reader" has an unclear description and is not localised. 

Note: to be decided, if this block should be moved to a new Accessibility section in the Settings.  

<img width="727" height="305" alt="Image" src="https://github.com/user-attachments/assets/5a89f0e8-cc2f-4f7f-8460-319d9234f944" />

**Comment by danielweck** [comment]
Related : https://github.com/edrlab/thorium-reader/issues/3349
